### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Paskab012**
Hello ✋🏼 ,
In this PR, I have implemented my project by using a relational database to create the initial data structure for a vet clinic
## MAIN OBJECTIVES
- Create tables in SQL.
- Insert and query data in SQL.
## REQUIREMENTS
- I have used [this template](https://github.com/microverseinc/curriculum-template-databases) to generate my repo.
- I have created a database named vetclinic.
- I have created a table named `animals` with the following columns:
  - `id: integer`
  - `name: string`
  - `date_of_birth: date`
  - `escape_attempts: integer`
  - `neutered: boolean`
  - `weight_kg: decimal`
 ## WRITING QUERIES
- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
## ANIMALS TABLE
Screen Shot 2022-05-06 at 11 34 15 AM

### CODE REVIEW

**Review by Kingstalux — COMMENTED** [review]
Hi @Paskab012 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966

## Invalid Code Review Request ❌ 

- [x] Kindly use the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Your pull request should be from your feature branch into the `development` branch. It would be great if you could rectify this issue 👍 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Paskab012** [comment]
Hello @Kingstalux , My apologies for this. I understood...

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,26 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT * from animals WHERE `date_of_birth` BETWEEN '2016-01-01'  AND '2019-01-01';
+
```
Comment (mahbubul14) [inline queries.sql]: - [x] it should be listed the name of all animals born between 2016 and 2019. Kindly check the following line to get an idea:
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    name varchar(50),
+    date_of_birth DATE,
+    escape_attemps INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL 
+);
```
Comment (mahbubul14) [inline schema.sql]: - [x] **[Required]** I think you should add an `id` column in the table.

- [x] **[OPTIONAL]** you can also mark the id column as the primary key.

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Paskab012 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @Paskab012 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**